src/runtime/virtcontainers/qemu.go

```diff
@@ -1088,6 +1178,13 @@ func (q *qemu) cleanupVM() error {
 			}).Debug("successfully removed the non root user")
 	}
 
+	if q.qemuCmd != nil {
+		if err := q.qemuCmd.Wait(); err != nil {
```

Comment:
Maybe I miss something here, but I thought Wait() will block forever until the QEMM process exists. In that case, don't you need to call Wait() in a separate goroutine?

Reply:
Yeah. Would it be better to call `Wait()` in the `stopVM` method instead of `cleanupVM`?

Reply:
After some more thoughts, I'll move Wait() back to govmm but put it in a goroutine.